src/class-tumblr-import.php

```diff
@@ -0,0 +1,1560 @@
+<?php
+/**
+ * Contains the main class for the Tumblr Importer.
+ */
+
+namespace Automattic\TumblrImporter;
```

Comment:
I think we should avoid using `Automattic` as the namespace since.

Reply:
Changed it for WordPress